docs/rest.md

```diff
@@ -610,14 +610,14 @@ The response is a stream of assertions separated by double newlines.
 The X-Ubuntu-Assertions-Count header is set to the number of
 returned assertions, 0 or more.
 
-## /2.0/skills
+## /2.0/interfaces
 
 ### GET
 
-* Description: Get all the skills and information about where they are granted
+* Description: Get all the plugs and slots and information about what is connected to them.
```

Comment:
Maybe something along the lines of __Get all the plugs and information about the slots connected to them__ would be more descriptive?